platforms/documentation/docs/src/docs/userguide/best-practices/best_practices_general.adoc

```diff
@@ -175,3 +175,92 @@ include::sample[dir="snippets/bestPractices/avoidInternal/groovy",files="build.g
 === Tags
 
 <<tags_reference.adoc#tag:upgrades,`#upgrades`>>
+
+[[use_the_gradle_properties_file]]
+== Set build flags in `gradle.properties`
+:keywords: properties,structuring-builds
```

Comment:
I am temporarily removing keywords due to the new search algorithm I just deployed. I will add them back if appropriate:
```suggestion
```